scripts/build/builders/tizen.py

```diff
@@ -49,11 +56,20 @@ class TizenApp(Enum):
         ('chip-tool',
          'chip-tool.map'))
 
+    TESTS = TestDriver(
+        'tests',
+        'src/test_driver/tizen')
+
     @property
     def is_tpk(self):
         """If True, this app is a TPK."""
         return isinstance(self.value, App)
 
+    @property
+    def is_test(self):
```

Comment:
how does this work? Trying to see usage of TESTS and is_test, yet I cannot find it.

Concretely I was wondering why pre-building binaries and enabling flash bundle is required. Asking for 'you need to build this before building this other thing' is not intuitive, generally just saying "run tests" should be sufficient.

Reply:
This is a left-over from some of my earlier experiments. It can be removed.